pkg/daemon/daemon.go

```diff
@@ -40,7 +40,7 @@ type Daemon struct {
 	Pid              int
 	ImageID          string
 	DaemonMode       string
-	apiSock          *string
+	APISock          *string
```

Comment:
We'd better add a hint for the field name in JSON format to avoid the case-sensitivity issue like this:

```
APISock *string `json:"APISock"`
```

Reply:
The json tag of the member variable is used to ensure persistence compatibility. When the variable name is modified, it can still maintain the consistency of the nydus db storage field name (of course, the first letter of the variable name should still be capitalized), so I think it is not just APISock need to add json tag, all fields can be considered to be added, so I will submit a separate patch to do this in the future.